client/app/host/QuestionAccordion.tsx

```diff
@@ -0,0 +1,96 @@
+'use client';
+import { Switch } from '@/components/ui/switch';
+import {
+    Accordion,
+    AccordionContent,
+    AccordionItem,
+    AccordionTrigger,
+} from '@/components/ui/accordion';
+
+interface QuestionAccordionProps {
+    questions: {
+        question: string;
+        description: string;
+        language: string;
+        points: string;
+        tests: {
+            input: string;
+            output: string;
+        }[];
+        enabled: boolean;
+    }[];
+    handleQuestionSwitch: (question: string) => void;
+}
+
+const QuestionAccordion: React.FC<QuestionAccordionProps> = ({
+    questions,
+    handleQuestionSwitch,
+}) => {
+    return (
+        <Accordion type="single" collapsible className="w-full pl-2 pr-2">
+            {questions.map((q, index) => (
+                <AccordionItem
+                    key={index}
+                    value={`question-${index}`}
+                    className={`px-2.5} mb-1 rounded border`}
+                >
+                    <AccordionTrigger
+                        className={`flex max-w-full px-1.5 ${q.enabled ? '' : 'bg-[#666a] opacity-50'}`}
+                    >
+                        <p className="w-2/3 truncate">
+                            {index + 1}. {q.question}
+                        </p>
+                        <p>{q.points} pts</p>
+                    </AccordionTrigger>
+                    <AccordionContent>
```

Comment:
Add `px-1.5` class here to align the content with the trigger

Reply:
I believe you added the `px-1.5` to the trigger.  It should go on the `AccordianContent` so that they are aligned:
![image](https://github.com/user-attachments/assets/2a5ff5a7-a173-45a8-a2ed-dbbd6d90ea39)
```suggestion
                    <AccordionContent className="px-1.5">
```

Reply:
I think it may look better to have the "3. " and "Convert" aligned, like in the image above 

vs the current: 
![image](https://github.com/user-attachments/assets/fae08504-3a5a-4448-bb5d-b4b2141dde97)

Reply:
I forgot to fix it in the last commit. Should be fixed now.